src/character/EvercraftCharacter.java

```diff
@@ -8,6 +8,15 @@
 	private Integer armorClass = 10;
 	private Integer hitPoints = 5;
 	private Integer strength = 10;
+	private Level level;
+	
+	public EvercraftCharacter(Level level) {
+		this.level = level;
+	}
```

Comment:
If we assume every created character begins at level 1 we can have each character create a new instance of level for their use. When can then just mock away the returning of the level as opposed to passing in a mock to the constructor